systemd/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+include(GNUInstallDirs)
+
+configure_file(${CMAKE_SOURCE_DIR}/systemd/ceph-osd@.service.in
+  ${CMAKE_CURRENT_BINARY_DIR}/ceph-osd@.service @ONLY)
+
+install(FILES
+  ${CMAKE_BINARY_DIR}/systemd/ceph-osd@.service
+  DESTINATION ${CMAKE_INSTALL_LIBEXECDIR}/systemd/system)
```

Comment:
```
install -m0644 systemd/ceph-osd@.service debian/ceph-osd/lib/systemd/system
install: cannot stat 'systemd/ceph-osd@.service': No such file or directory
debian/rules:55: recipe for target 'override_dh_installinit' failed
make[1]: *** [override_dh_installinit] Error 1
```

see https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=xenial,DIST=xenial,MACHINE_SIZE=huge/1297//consoleFull

as we are now collecting the .service files manually in the `debian/rules`. and editing it on the fly:
```sh
sed -i s./etc/sysconfig/./etc/default/.g debian/ceph-osd/lib/systemd/system/ceph-osd@.service
```
as fedora/RHEL/SuSE uses `/etc/sysconfig` while debian uses `/etc/default`. maybe we could make the prefix a subvar and configure it in `ceph.spec` and `debian/rules` or stick to current approach. but either way we will be forced to reference the file in the build directory (`$(DESTDIR)`).

not sure if it's worthy to go this far, though...